cmd/virt-api/Dockerfile

```diff
@@ -18,11 +18,17 @@
 
 FROM centos:7
 
+# Create non-root user
+RUN useradd --create-home -s /bin/bash virt-api
+WORKDIR /home/virt-api
+USER virt-api
```

Comment:
RUN useradd - create new user
USER virt-api - change execution user to virt-api, so all commands after this one will be executed with user virt-api
so if you want to include
securityContext:
        runAsNonRoot: true
to the manifest yaml you musr run this command in the dockerfile

I believe I can include this lines in the end of the Dockerfile and do not change all swagger commands(I believe I will do it this way for the haproxy container), give me know if you prefer it this way